src/scss/_home.scss

```diff
@@ -1,13 +1,12 @@
 $home-color: #eee;
-$home-color-yellow: rgb(250, 162, 27);
 
-[class=home] {
+.home {
   background-attachment: fixed;
   background-image: url('../img/background.png');
   background-position: center;
   background-repeat: no-repeat;
   background-size: cover;
-  color: $home-color;
+  color: $white-text;

```

Comment:
$akai-white?

Reply:
White text is not quite white, it's #eee. That's why we have two variables.